crates/levm/src/vm.rs

```diff
@@ -452,14 +471,105 @@ impl VM {
                         .memory
                         .copy(src_offset, dest_offset, size);
                 }
+                Opcode::CALL => {
+                    let gas = current_call_frame.stack.pop().unwrap();
+                    let address =
+                        Address::from_low_u64_be(current_call_frame.stack.pop().unwrap().low_u64());
+                    let value = current_call_frame.stack.pop().unwrap();
+                    let args_offset = current_call_frame.stack.pop().unwrap().try_into().unwrap();
+                    let args_size = current_call_frame.stack.pop().unwrap().try_into().unwrap();
+                    let ret_offset = current_call_frame.stack.pop().unwrap().try_into().unwrap();
+                    let ret_size = current_call_frame.stack.pop().unwrap().try_into().unwrap();
+
+                    // check balance
+                    if self.balance(&current_call_frame.msg_sender) < value {
+                        current_call_frame.stack.push(U256::zero());
+                        continue;
+                    }
+
+                    // transfer value
+                    // transfer(&current_call_frame.msg_sender, &address, value);
+
+                    let callee_bytecode = self.get_account_bytecode(&address);
+
+                    if callee_bytecode.is_empty() {
+                        current_call_frame.stack.push(U256::one());
```

Comment:
I think is just me, but the return of the `Call` opcode always confuse me, maybe we could have some constants like `SUCCESS_FOR_CALL` and `REVERT_FOR_CALL`, if you don't think is neccesary is also fine